hubblestack/extmods/modules/pulsar.py

```diff
@@ -395,3 +395,44 @@ def _dict_update(dest, upd, recursive_update=True, merge_lists=False):
             for k in upd:
                 dest[k] = upd[k]
         return dest
+
+#TODO:change paths for top.pulsar and config yaml to 'salt://hubblestack_pulsar/top.file'
+
+def top(topfile='/root/myhubblefiles/topfile/top.pulsar',
+        debug=None):
+
+    results_pulsar = []
+
+    configs = get_top_data(topfile)
+
+    if not isinstance(configs, list):
+        configs = configs.split(',')
+
+    configs = ['/root/myhubblefiles/topfile/' + config + '.yaml'
+               for config in configs]
+
+    results_pulsar.extend(process(configs, verbose=False))
```

Comment:
I think here, since `results_pulsar` is empty, you can just do a straight return:

```
return process(configs, verbose=verbose)
```

Note that I have `verbose=verbose` -- the `top()` function should take a verbose argument and pass it through to `process()`.